benchmarks/python/test_gelu_bwd.py

```diff
@@ -109,6 +111,9 @@ def gelu_fwd():
     fwd_fn = {
         "eager": gelu_fwd,
         "torchcompile": torch.compile(gelu_fwd),
+        "thunder": thunder.jit(
+            gelu_fwd, executors=[nvfuserex], nv_enable_bookend=False
+        ),
     }
     outputs = fwd_fn[executor]()
```

Comment:
Maybe we should abstract this away with a couple new functions in `core.py`. For example instead of listing the executors directly in we could use
```python
@pytest.mark.parametrize('executor', default_executors)
```
Then down here we could use
```python
fwd_fn = with_executor(executor, gelu_fwd)
```
That way we can enable/disable executors in a centralized location.